### PR TITLE
native worker output

### PR DESCRIPTION
**Pull request by michaelglass**
moves our API closer to rspec's actual API and allows for real RSpec test output on the workers

### CODE REVIEW

File: lib/rspec/abq/formatter.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
+require "time"
```
Comment (michaelglass) [inline lib/rspec/abq/formatter.rb:1]: this is unecessary now

**Review by tonywok — APPROVED** [review]
Your comments were super helpful.

**Comment by michaelglass** [comment]
thanks for the helpful nudge, `bin/check_yard`

File: lib/rspec/abq/formatter.rb (hunk 1 above)
Comment (michaelglass) [reply lib/rspec/abq/formatter.rb:1]: actually it's used for the time output below! nevermind

**Pull request merged**